source/wormhole-strategy.hpp

```diff
@@ -0,0 +1,24 @@
+/* wormhole-strategy.hpp
+Copyright (c) 2022 by warp-core
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#ifndef WORMHOLE_STRATEGY_HPP_
+#define WORMHOLE_STRATEGY_HPP_
+
+#include <cstdint>
+
+enum class WormholeStrategy : int_fast8_t {
```

Comment:
Can't this be put in DistanceMap.h? It's also used there so I don't really see the need in having an additional header for 4 liens of code.

Reply:
https://github.com/endless-sky/endless-sky/pull/6846#discussion_r885095864